### DISCUSSION TITLE
The From address is not authorised to send mail from this server

### DISCUSSION THREAD

**Discussion by Edgar-aa** [postalserver/postal#2094] [Help with using Postal] [answered]
Please Help what will i do ,show where will i add config for fix this.

{
    "status": "error",
    "time": 0.01,
    "flags": {},
    "data": {
        "code": "UnauthenticatedFromAddress",
        "message": "The From address is not authorised to send mail from this server"
    }
}

**Comment by willpower232** [comment]
> The From address is not authorised to send mail from this server

The domain name you're sending from needs to be in the same server as the credential within the Postal UI.

**Reply by Edgar-aa** [reply]
as api credential or smtp?

**Reply by Edgar-aa** [reply]
example i have mail.test.com server 
emails begin whit @test.com
i try to send email from admin@test.com to admin@test.com .
api credential name will be admin@test.com ?

**Reply by willpower232** [reply]
either, if you can't see the domain on the Domains tab after you copy the credential from the Credentials tab then it won't work

**Reply by Edgar-aa** [reply]
i see domain ,but its not work

**Comment by Edgar-aa** [comment, marked as answer]
you need to write in the instructions that "Send a raw RFC2882 message
/api/v1/send/raw" doesn't work!
so that they do not break over it in vain

**Reply by willpower232** [reply]
when using the raw api, you need to make sure that the domain used in the "mail_from" parameter is also used as a From: header in the message you are sending

**Reply by PixeloDigital** [reply]
can you send one example as I have tried.
{
    "From" : "user2@example.in",
    "mail_from" :"user2@example.in",
    "rcpt_to" : ["user@example.com"],
    "data" : "the base64 encoded string from the output of the raw message payload"
}

In Headers:
X-Server-API-Key : xxxxx

**Reply by willpower232** [reply]
I was referring to a header in the message so it will be part of the `data` attribute in the base64 encoded value.

**Comment by phillmorgan28** [comment]
Reopening as I'm a little confused. Does this endpoint work or not?
